docker-compose.uffizzi.yml

```diff
@@ -0,0 +1,52 @@
+version: "3"
+
+x-uffizzi:
+    ingress:
+        service: nginx
+        port: 80
+
+services:
+    server:
+        image: "${PADLOC_SERVER_IMAGE}"
+        environment:
+            PL_DATA_BACKEND: leveldb
+            PL_DATA_LEVELDB_DIR: /data
+            PL_DATA_ATTACHMENTS_BACKEND: fs
+            PL_DATA_ATTACHMENTS_DIR: /attachments
+            PL_EMAIL_BACKEND: console
+        expose:
+            - 3000
+        volumes:
+            - attachments:/attachments
+            - data:/data
+        restart: unless-stopped
+        deploy:
+          resources:
+            limits:
+              memory: 4000M
+
+    pwa:
+        image: "${PADLOC_PWA_IMAGE}"
+        environment:
+            PL_PWA_PORT: 80
```

Comment:
Is there no way to get the preview URL here to set `PL_PWA_URL=$UFFIZZI_URL`?  Otherwise you'll run into CSP issues. I'm working on adding an option to disable it in #637 but that's not finalized just yet.

Reply:
I am using nginx and getting rid of the CSP issue right now. We are working on the method you mentioned [here](https://github.com/UffizziCloud/uffizzi/issues/261).

Reply:
`PL_PWA_URL=$UFFIZZI_URL` setting this in the docker-compose now. Marking this review as resolved.